### DISCUSSION TITLE
I have issue when run next-sitemap to building sitemaps

### DISCUSSION THREAD

**Discussion by vudq291295** [iamvishnusankar/next-sitemap#781] [General]
i have error :
 ❌ [next-sitemap] Unable to find build-manifest.
Make sure to build the project using `next build` command

 ❌ [next-sitemap] undefined
 
 
![image](https://github.com/iamvishnusankar/next-sitemap/assets/15715702/41140409-d937-47a6-ad0d-0742bb880da9)

**Comment by hakimov-dev** [comment]
Have u find solution for this?
